### DISCUSSION TITLE
Make it so it always finds the most recent version of AE

### DISCUSSION THREAD

**Discussion by JamesBotterill** [inlife/nexrender#832] [Ideas]
At present when the nexrender is looking for a version of AE to render against and there's multiple versions installed on the rendering machine it will default to the oldest version installed. 

This isn't idea in a number of scenario's so I wondered if it's worth making this more flexible.  

My ideal solution would be to have a setting in the set up similar to binary where you can define "oldest" or "newest" and then filter the search compared to that. 

While binary might serve this purpose the issue is that it requires you know the absolute path to the binary which will not be possible in the application I'm working on.

**Comment by inlife** [comment]
Actually, I think even the newest by default would be a great option, especially considering that autofind works by just going over every case one by one, and reversing that order would implement the suggestion!

https://github.com/inlife/nexrender/blob/master/packages/nexrender-core/src/helpers/autofind.js

However, that would be a breaking change, and being the responsible fella I try to be, it would be an ill-advised decision to do that by default. 😄  However, we could implement a flag/setting that would change the sorting order and consider that to be an optimal solution.

What do you think?

**Reply by JamesBotterill** [reply]
As it's a breaking change and there's an advantage to keeping the current method of oldest first as you don't want to might want migrate a project to the latest version and validating it's still working prior to switching to latest version

I believe having it as an flag such as below would work with the default being oldest, to not introduce a breaking change

`--ae-history=newest`

**Reply by inlife** [reply]
@JamesBotterill I think --ae-history parameter might be better called something like:
* `--ae-version=@latest`
* `--ae-version=@oldest`
* `--ae-version=2023`
* `--ae-version=v12.02`

What do you think?

**Reply by JamesBotterill** [reply]
yeah that's perfect

Would it be possible to set a minimum version so `--ae-version=>2023 `

**Reply by inlife** [reply]
I think it is in our power, so yeah
